apollo-federation/src/sources/connect/json_selection/snapshots/apollo_federation__sources__connect__json_selection__parser__tests__error_snapshots.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: apollo-federation/src/sources/connect/json_selection/parser.rs
+expression: "JSONSelection::parse(\".data\")"
+---
+Err(
+    Failure(
+        JSONSelectionParseError {
+            message: "Key paths may no longer start with just .key (use $.key instead)",
+            fragment: ".data",
+            offset: 0,
+        },
+    ),
+)
```

Comment:
This error message should help explain/mitigate the breaking change introduced by #6086.

Reply:
"no longer" isn't going to make much sense to folks not involved with the preview (so: everyone 😁 )

Reply:
Fair enough: https://github.com/apollographql/router/pull/6089/commits/39da23563041cd2c857dda808aaae8c9d733f7d2